tdrs-backend/tdpservice/settings/common.py

```diff
@@ -110,6 +75,10 @@ class Common(Configuration):
     WSGI_APPLICATION = "tdpservice.wsgi.application"
     CORS_ORIGIN_ALLOW_ALL = True
 
+    # Application URLs
+    BASE_URL = os.getenv('BASE_URL', 'http://localhost:8080/v1')
```

Comment:
It's important to note that although there are defaults supplied for most of these settings, setting the listed environment variable will take precedence over the default, allowing us to fine tune configuration as needed for different environments.